docs/content/contributing/_index.en.md

```diff
@@ -58,7 +58,7 @@ $ helm dep update $GOPATH/src/github.com/fission/fission/charts/fission-all
 Next, install fission with this image on your kubernetes cluster using the helm chart:
 
 ```sh
-$ helm install --set "image=minikube/fission-bundle,pullPolicy=IfNotPresent,analytics=false" charts/fission-all
+$ helm install --set "image=minikube/fission-bundle,imageTag=latest,pullPolicy=IfNotPresent,analytics=false" charts/fission-all
```

Comment:
How about changing L49 to
```
$ docker build -t minikube/fission-bundle:<tag> -f cmd/fission-bundle/Dockerfile.fission-bundle .
```
and then
```
helm install --set "image=minikube/fission-bundle,imageTag=<tag>.....
```

Reply:
Thank you for the comment. Your comment makes sense but would that not mean the use who would be just copying and pasting the command have to input the tag ?

Or maybe, we can add a static tag like,

```
$ docker build -t minikube/fission-bundle:1.0 -f cmd/fission-bundle/Dockerfile.fission-bundle .
```
and

```
$ helm install --set "image=minikube/fission-bundle,imageTag=1.0.....

```

I am not sure how much sense tagging it as 1.0 would make though. 

Any reason we are tagging it instead of just leaving it as is ?

Reply:
The original idea of the comment is that if we don't assign the `imageTag` kubernetes will use `latest` when pulling image automatically. If we want to add `imageTag` to install command, isn't it better to add `<tag>` instead of 

For my personal workflow, I will do something like

```
$ docker build -t minikube/fission-bundle:pr1234 ....
$ helm install --set "image=minikube/fission-bundle,imageTag=pr1234
```

What do you think?

Reply:
Makes sense. We might have to add a comment below telling the user to replace the <tag> with the tag they want or just use 'latest' as the tag if they don't want to think about which tag they want to use.

What do you think ?

Reply:
ok, that's fair.